### ISSUE TITLE
Sind die Größen der Werbebanner bereits festgelegt?

### ISSUE THREAD

**Issue by Revanjin** [th-koeln/mi-bachelor-gdvk#15]
Sind diese Festgelegt oder gibt es in dem Bereich gegebenenfalls Anpassungsmöglichkeiten bzgl des Designs?

**Comment by lliane** [comment]
Ja. Im Grunde kannst du genau die verwenden, die du bereits jetzt auf der Seite findest. 

Skyscaper: 160 x 600 px
https://monosnap.com/file/gSrmMo2H2DyFalUyICHl2sDAeGEFCo

Banner: 468 x 60 px
https://monosnap.com/file/g0m1R6LMVEZ6dJSIDb9ARMIxjdscHW

Google Ads: 
https://monosnap.com/file/Zp41v9qX5Wdho8yRp5HrTzR5A5S6qF
Hier kannst du auf jeden Fall die Hintergrundfarbe ändern und sicher auch die Anzahl der angezeigten Blöcke variieren. Ich versuch dazu mal mehr Infos zu bekommen.

Hilft dir das erstmal?

Lieben Gruß,
Liane

**Comment by JulianSchoem** [comment]
Gibt es hier neue Informationen? Die Größen können wir frei wählen, oder?

**Comment by vschaefer** [comment]
Die Größen könnt ihr frei wählen. Die Banner sollten nur nicht verzerrt oder beschnitten werden. Das heißt die Seitenverhältnisse sollten beibehalten werden. Zudem sollten die Banner nicht zu klein sein, so dass sie unleserlich werden.
